apps/i18n/common/pl_pl.json

```diff
@@ -1535,6 +1535,8 @@
   "unpluggedActivity": "Aktywność unplugged (bez komputera)",
   "unpublish": "Cofnij publikację",
   "unpublishPending": "Usuwanie publikacji...",
+  "unpublishFailureTitle": "Cofnięcie publikacji nie powiodło się",
+  "unpublishFailureBody": "Nie udało się cofnąć publikacji Twojej biblioteki „{nazwa_biblioteki]”. Sprawdź połączenie internetowe i spróbuj ponownie.",
```

Comment:
FYI @castro-jorge that a translator translated a variable https://crowdin.com/translate/codeorg/43/enus-pl#2448280